prow/config/tide.go

```diff
@@ -444,7 +444,7 @@ func (c Config) GetTideContextPolicy(org, repo, branch string) (*TideContextPoli
 			logrus.WithError(err).Warningf("Error getting branch protection for %s/%s+%s", org, repo, branch)
 		} else if bp == nil {
 			logrus.Warningf("branch protection not set for %s/%s+%s", org, repo, branch)
-		} else if bp.Protect != nil && *bp.Protect {
+		} else if bp.Protect != nil && *bp.Protect && bp.RequiredStatusChecks != nil {
```

Comment:
@cjwagner is this how we want to handle this? What about if all the contexts are optional but we still want the required reivew etc

Reply:
Tide here is only worried about the contexts, though -- not the rest of it. PR in unmergable state is handled elsewhere